msg/CMakeLists.txt

```diff
@@ -104,6 +105,7 @@ set(msg_files
 	manual_control_setpoint.msg
 	manual_control_switches.msg
 	mavlink_log.msg
+	mavlink_tunnel.msg
```

Comment:
@roman-dvorak This also seems very unlrelated to this PR, better remove it

Reply:
For this improvement, PR is ready: https://github.com/PX4/PX4-Autopilot/pull/16408

But after all the adjustments were resolved, no one answered me (or no one merged it). And it looks like it's still mergeable.